### DISCUSSION TITLE
github wiki features?

### DISCUSSION THREAD

**Discussion by cru121** [community/community#21744] [New to GitHub]
<p>Does the github wiki support:</p>
<p>* Templates</p>
<p>* Categories</p>
<p>* Changing text color</p>

**Comment by Gisgar3** [comment, marked as answer]
<p><strong>GitHub Wiki</strong> allows for many benefits, _especially _for documentations, release information, and much much more! Categories in  <strong>GitHub Wikis</strong> are best defined as the <em>sidebar</em>; given its name, the <em>sidebar</em> appears on the right-side of a  <strong>GitHub Wiki</strong> and can be customly formatted via  <strong>Markdown</strong>. For templates (regarding the physical attributes of a wiki page), I have not seen any templates that can be added that will physically change the appearance of a wiki page; not sure if  <strong>GitHub</strong> officially supports wiki templates in the first place. For changing text color, wiki pages can only use the standard font and color that are provided with the page; no custom fonts or text colors can be added to change the pages appearance. Hope this helps!</p>
